app/src/main/java/io/github/droidkaigi/confsched2017/model/Speaker.java

```diff
@@ -34,4 +39,20 @@
     @Nullable
     @SerializedName("github_name")
     public String githubName;
+
+    @Nullable
+    public String getAdjustedImageUrl() {
+        if (imageUrl == null) {
+            return null;
+        }
+
+        if (imageUrl.startsWith("http://") || imageUrl.startsWith("https://")) {
```

Comment:
@k-kagurazaka Thanks! I know what you mean. But if I use `URLUtil` class, I also use Robolectric in the test code. As you said, it's not big problem, but I want to avoid it.